docs/getting-started.md

```diff
@@ -185,17 +185,37 @@ After each change, you can continue updating your component by using: `odo push
 
 ### 6. Adding storage to the component
 
-Now that you've got your component running, how do you add persistent any data between restarts?
+Now that you've got your component running, how do you persist data between restarts?
 
 If you wish to add storage to your component, `odo` makes it very easy for you to do this:
 
 ```console
 $ odo storage create nodestorage --path=/opt/app-root/src/storage/ --size=1Gi 
- OK  Added storage nodestorage to nodejs-ex-nodejs-nnjf
+ ✓   Added storage nodestorage to nodejs-ex-nodejs-nnjf
 ```
 
 That's it! Storage has been added your component with an allocated size of 1 Gb.
 
+## Command completion
+
+**Command completion is currently only supported for `bash`, `zsh` and `fish` shells.**
+
+`odo` provides smart completion of command parameters based on user input. For this to work, `odo` needs to integrate with the 
+executing shell. This is accomplished by running `odo --complete` and pressing `y` when asked to install the completion hook. 
+This modifies your shell configuration file (e.g. `.bashrc` for `bash`).
+
+You can also modify your shell configuration file manually by adding:
+```sh
+complete -o nospace -C <full path to your odo binary> odo
+``` 
+
+To disable completion, run `odo --uncomplete`.
+
+After any modification to your shell configuration file, you will need to `source` it or restart your shell.
+
+**NOTE**: The completion system will stop working if you either rename the `odo` executable or move it and you will to re-enable
```

Comment:
`The completion system will stop working if you either rename the `odo` executable or move it. The you will need to re-enable`

Reply:
Improved.